### ISSUE TITLE
PrettyPrinter removes newlines

### ISSUE THREAD

**Issue by biswanaths** [biswanaths/scala_jira_gh#102]
<div class="user-content-block">
<p>=== What steps will reproduce the problem (please be specific and </p>
<div style="margin-left: 1em;"><pre class="brush: scala; gutter : false; toolbar: false;">
scala&gt; &lt;foo&gt;{PCData("hello\nworld")}&lt;/foo&gt;
res51: scala.xml.Elem =
&lt;foo&gt;&lt;![CDATA[hello
world]]&gt;&lt;/foo&gt;

scala&gt; (new PrettyPrinter(Int.MaxValue, 2)).format(res51)
res52: String = &lt;foo&gt;&lt;![CDATA[hello world]]&gt;&lt;/foo&gt;
</pre></div>

<p>=== What is the expected behavior? ===<br/>
Should print "hello\nworld"</p>

<div style="margin-left: 1em;"><pre class="brush: scala; gutter : false; toolbar: false;">
scala&gt; (new PrettyPrinter(Int.MaxValue, 2)).format(res51)
res52: String = &lt;foo&gt;&lt;![CDATA[hello\nworld]]&gt;&lt;/foo&gt;
</pre></div>

<p>=== What do you see instead? ===<br/>
prints "hello world"</p>

<p>=== What versions of the following are you using? ===</p>

<ul class="alternate" type="square">
<li>Scala: 2.8.1</li>
</ul>

<a href="https://issues.scala-lang.org/browse/SI-4543">Imported from : SI-4543</a></div>

**Comment by som-snytt** [comment]
@biswanaths Are you aware of the JIRA spam?

**Comment by biswanaths** [comment]
@som-snytt , I assume you are talking about https://issues.scala-lang.org/browse/TEST-14

Sorry for the inconvenience caused by barrage of test/spam mail. I assumed the test project is not used and watched by anybody and I needed a ticket on which to test the migration script I was writing for pulling out xml jira issues from jira to scala-xml. 

If you are not talking about Test-14, please let me know what else is the issue.

**Comment by som-snytt** [comment]
Yes, that's it. I was just checking that something wasn't running unattended. :)
